e2e/cypress/integration/notifications/ignore_channel_mentions.js

```diff
@@ -0,0 +1,132 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @notifications
+
+function addNumberOfUsersToChannel(num = 1) {
+    // # Then click it to access the drop-down menu
+    cy.get('#channelHeaderTitle').click();
+
+    // * The dropdown menu of the channel header should be visible;
+    cy.get('#channelLeaveChannel').should('be.visible');
+
+    // # Click 'Add Members'
+    cy.get('#channelAddMembers').click();
+
+    // * Assert that modal appears
+    // # Click the first row for a number of times
+    Cypress._.times(num, () => {
+        cy.get('#multiSelectList').should('be.visible').children().first().click();
+    });
+
+    // # Click the button "Add" to add user to a channel
+    cy.get('#saveItems').click();
+
+    // # Wait for the modal to disappear
+    cy.get('#addUsersToChannelModal').should('not.exist');
+}
+
+function setIgnoreMentions(toSet) {
```

Comment:
nit: can we make this receive a boolean instead so we don't have to write the 'On'/'Off' string constants?